sormas-backend/src/main/java/de/symeda/sormas/backend/util/JurisdictionHelper.java

```diff
@@ -262,4 +263,23 @@ public static EventParticipantJurisdictionDto createEventParticipantJurisdiction
 
 		return eventParticipantJurisdiction;
 	}
+
+	public static JurisdictionLevel getSuperordinateJurisdiction(JurisdictionLevel jurisdition) {
+		switch (jurisdition) {
+		case NATION:
+			return JurisdictionLevel.NONE;
+		case REGION:
+			return JurisdictionLevel.NATION;
+		case DISTRICT:
+			return JurisdictionLevel.REGION;
+		case COMMUNITY:
+		case POINT_OF_ENTRY:
+		case HEALTH_FACILITY:
+		case LABORATORY:
```

Comment:
Region and district aren't mandatory for laboratories, so I'd rather give them jurisdiction level NONE; external laboratories are supposed to be outside the country, so they should also receive NONE